CHANGES.md

```diff
@@ -66,6 +66,9 @@
 - virtual libraries: fix an issue where linking an executable involving several
   virtual libries would cause an error. (#10581, fixes #10460, @rgrinberg)
 
+- Correctly declare dependencies of indexes so that they are rebuilt when
+  needed. (#10623, @voodoos)
+
```

Comment:
this is a bit messy, but this is an entry of alpha2, not alpha1. we'll collapse these at release time (and this entry will be removed since it's not relevant to released versions)

Reply:
ah, didn't realize that we keep them separate. Fixed.